src/lib/profiles/security/WeaveCASEEngine.cpp

```diff
@@ -1006,6 +1016,176 @@ bool WeaveCASEEngine::IsAllowedConfig(uint32_t config) const
 }
 
 
+#if !WEAVE_CONFIG_LEGACY_CASE_AUTH_DELEGATE
+
+WEAVE_ERROR WeaveCASEAuthDelegate::EncodeNodePayload(const BeginSessionContext & msgCtx,
+            uint8_t * payloadBuf, uint16_t payloadBufSize, uint16_t & payloadLen)
+{
+    payloadLen = 0;
+    return WEAVE_NO_ERROR;
+}
+
+WEAVE_ERROR WeaveCASEAuthDelegate::OnPeerCertsLoaded(const BeginSessionContext & msgCtx,
```

Comment:
should it be inline function?

Reply:
No.  This is a virtual function, so it should be non-inline.